runtime/compiler/trj9/env/j9method.cpp

```diff
@@ -4504,7 +4505,19 @@ TR_ResolvedJ9Method::TR_ResolvedJ9Method(TR_OpaqueMethodBlock * aMethod, TR_Fron
          else if ((classNameLen == 29) && !strncmp(className, "java/lang/invoke/MethodHandle", 29))
             {
             if (!strncmp(name, "asType", 6))
-               setRecognizedMethodInfo(TR::java_lang_invoke_MethodHandle_asType);
+               {
+               int32_t instanceAsTypeSigLen = sizeof("(Ljava/lang/invoke/MethodType;)Ljava/lang/invoke/MethodHandle;");
```

Comment:
I don't believe signatures from the VM are guaranteed to have a '\0' so does this actually work in practice? I think it may read one character too many...

Reply:
Good catch, I'll change it to `strlen`.